### ISSUE TITLE
[Proposal]: Daily crew costs

### ISSUE THREAD

**Issue by clintonmead** [naev/naev#2550]
### Suggestion

Daily crew costs, even if quite nominal, could introduce some reasonable gameplay compromises. Currently, if one isn't on a timed mission, one can completely ignore jump and landing time, stop at every station on the way without penalty, even in a huge ship with a large crew. With daily crew costs, one might think twice meandering around in a ridiculously large ship with a hundred crew members. Which makes sense. If one wants to just travel around on a holiday and explore and see the sights, it would be more reasonable to do this on a ship with minimal crew requirements. 

These crew costs should be barely noticeable in the early game on relatively fast ships, but on slow large ships they may start to be significant. Not so much that they bankrupt a player just playing the game, but just that should have to do at least some basic missions on their travels relative to the size of their ship to finance themselves, and think twice about stopping at every station on the way.

The question the becomes what happens if the player's balance goes negative, and I'm not sure what the best answer to that is, but there are lots of options. 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bobbens** [comment]
Honestly, I don't think crew management is really something that makes sense in the base Naev game. The way the game is centered and focused on is player-centric, and adding more micromanagement and stuff does seem to move away from the more rpg / story-focus there is now. You may want to take a look at the #2152  PR that will not get merged on another direction that can be taken. I think it should be best left as a plugin.

The main way I could imagine "crew" making sense in Naev is as a mutually exclusive intrinsic outfit that could give some minor bonuses to a ship. Somewhat like the combat AIs or hull types. I briefly mention that in #2032 . However, I'm not sure it's really necessary, and I think just adding other types of intrinsic outfits that don't explicitly refer to crew would probably be a better approach for now.

**Comment by clintonmead** [comment]
I'm not suggesting crew management here, just a fixed cost per day per crew member in the fleet. It will just be automatically deducted at the daily tick. But otherwise, all crew will be the same. And there will no need to recruit/dismiss them.

I'd be happy if I could just have a plugin where one could set a value "crew cost per day" that's just deducted at the end of the day tick.

**Comment by bobbens** [comment]
Yes, that would be fine as long as it doesn't display anything and is completely hidden when not enabled, so without the plugin nothing changes with the current game.

That said, I'm not sure exactly where this would have to be displayed. I guess the info window, but probably the landing window too? Maybe the GUI? Implementing that in a flexible way could add a bit of development overhead if this has to be maintained in main Naev. I would be partial to an implementation that has minimal overhead.

A potential implementation without adding new C could would be to have an event that hooks time changes, and then makes the player pay. This could be accessible by adding a new button to the info window (doable via Lua), and wouldn't be too visible, but would work already without much fuss.

**Comment by clintonmead** [comment]
Perhaps this could be done entirely via a plugin. Money just needs to be deducted per day, based on the total crew of the fleet multiplied by a fix costs, perhaps with a message being displayed to the communication log. But I see no need to change the UI anywhere else.

**Comment by bobbens** [comment]
Yes, then it could be simply done by an event. Hook landing / takeoff / jumping + time changes, determine what ships the player is in, if they should pay, and how much they should pay. Can toss a message in the screen GUI and also have a log entry. Probably would make sense to also add some explanation by the ship AI or the likes somewhere.
